examples/thermomechanics/thermal_deformation.cpp

```diff
@@ -61,15 +61,16 @@ void thermalDeformationExample( const std::string filename )
     //                Force model type
     // ====================================================
     using model_type = CabanaPD::PMB;
-    using thermal_type = CabanaPD::TemperatureDependent;
+    using thermal_type = CabanaPD::DynamicTemperature;
```

Comment:
Are we modifying the thermal_deformation problem? My idea was to keep this one and the thermal crack as is and add new examples with the actual heat transfer.

Reply:
Up to you - would it be the same problem, but with heat transfer or a different problem? 

Exactly this distinction - is there heat transfer or not?

Reply:
Thanks. Initially, I am thinking the same exact problems to make things easier. We may create new problems later on.